test/user.js

```diff
@@ -0,0 +1,106 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('./../app/models').user,
+  should = chai.should();
+
+describe('/users POST', () => {
+  it('Successful', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'firstName',
+        lastName: 'lastName',
+        username: 'username',
+        password: 'password',
+        email: 'email1@wolox.com.ar'
+      })
+      .then(res => {
+        res.should.have.status(201);
```

Comment:
we need to also check that the response body is what we expect and that the user is really created in the DB.